replacy/__init__.py

```diff
@@ -157,12 +183,25 @@ def cb(matcher, doc, i, matches):
 
             span._.description = self.match_dict[match_name].get("description", "")
             span._.category = self.match_dict[match_name].get("category", "")
+            span._.subcategory = self.match_dict[match_name].get("subcategory", "")
+            span._.suggestions_separator = self.match_dict[match_name].get("suggestions_separator", " ")
+            span._.construct_suggestion_function = self.match_dict[match_name].get("construct_suggestion_function", "")
```

Comment:
I've been meaning to remove `description` and `category` and feel weird adding this... does `replacy/util.py:get_novel_prop_defaults` not handle these?

Reply:
`get_novel_prop_defaults` handle this if they are available in match_dict, which is not the case of every NLP service, currently `suggestions_separator` and `construct_suggestion_function` are available in `writing-style` only

Reply:
are they used in writing style only? or are they needed for replaCy itself to function? if they are only used in a downstream service, why not set those values in that service?

Reply:
maybe I am confused because you say " this is needed in order for replacy/__init__.py code to work" but I don't see a reference to them in there?